src/ralph/networks/tests/test_models.py

```diff
@@ -339,6 +341,28 @@ def test_issue_next_hostname(self):
         self.assertEqual(ne.issue_next_free_hostname(), 's12300005.dc.local')
         self.assertEqual(ne3.issue_next_free_hostname(), 's100001.dc.local')
 
+    def test_issue_next_hostname_hostname_taken(self):
```

Comment:
@xliiv would test_issue_next_hostname_when_a_few_next_hostnames_are_taken be better?